unixfs/src/walk.rs

```diff
@@ -356,93 +446,21 @@ impl Walker {
 
 /// Represents what the `Walker` is currently looking at. Converted to `Entry` for public API.
 #[derive(Debug)]
-struct InnerEntry {
+pub(crate) struct InnerEntry {
```

Comment:
```suggestion
struct InnerEntry {
```

This doesn't seem like a required change.